src/main/java/core/basesyntax/service/impl/WriterServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriterService;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class WriterServiceImpl implements WriterService {
+
+    private static final int LINE_OFFSET = 1;
```

Comment:
why do we need that?

Reply:
student left me a comment that it was a magic number 😅 
I'll change it back)